### ISSUE TITLE
Table rendering in REPL does not have high enough contrast.

### ISSUE THREAD

**Issue by jswrenn** [brownplt/code.pyret.org#185]
The shading totally disappears when CPO is displayed on a projector.

**Comment by schanzer** [comment]
If it doesn’t work for *sighted* users in classrooms, this feels more like a usability issue than an accessibility one.

If our default CSS fails to work in this setting, it means we need better CSS

**Comment by schanzer** [comment]
You’re going to hate me for saying this, but all I can say is “we should hold ourselves to a higher standard.” We could alternate shadings between odd and even number rows. We could use wider borders, darker borders, or whatever — either way, I just can’t believe we can’t come up with something that works for projectors.

**Comment by blerner** [comment]
Nonsequitur. We *do* alternate shading on each row, and we do have borders. Etc. What we want to avoid is that the backgrounds of the rows obscure the contents of the rows, especially when the rows contain images with arbitrary background colors themselves... So we're limited in what palettes we can use.

Imho, this is the same as the problem that PowerPoint color schemes look good on either your own screen or the projector screen, but almost never both :-\ I think it's probably a good idea to consider having two separate color schemes, as a result.

**Comment by schanzer** [comment]
We alternate them already? Wow, it's barely visible on my own monitor (an admittedly-old Apple 27" cinema display).  The fact that I didn't realize this until now suggests that our current solution may not be as good as it could be.

I hear you on not wanting to hide the contents of the table, but as long as we have alternating rows, we run the risk of obscuring the table contents for two specific shades. Making one of those shades darker doesn't increase that risk, does it?

I think our use-case is _far_ simpler than PPTs, and that we should be able to do a better job here. Why not have the even rows just be slightly darker? Some simple experimenting in the Web Inspector gets the attached screenshot, and all I had to do was darken the even rows a few shades. While I was at it, I also made the header row much darker and more visually-distinct from data rows, which I think we need anyway. 

![image](https://user-images.githubusercontent.com/77126/42464602-90282d26-8377-11e8-9c18-327df2f95bcb.png)

**Comment by schanzer** [comment]
FYI, if you like the screenshot above, this is just two CSS file changes:

editor.css line 1690: #e2e2e2
editor.css line 1664: #666

**Comment by schanzer** [comment]
@jpolitz mind if I tackle this in a tiny PR (or naked commit)?

**Comment by blerner** [comment]
Don't be _too_ tiny -- keep in mind you need to double-check all the themes we currently support.
